AnnoyancesFilter/Popups/sections/subscriptions_specific.txt

```diff
@@ -34,6 +34,7 @@ turkuazgazetesi.net,yenialanya.com,mansetaydin.com,seskocaeli.com,kocaelibarisga
 !
 ! SECTION: Subscriptions - Regular rules
 !
+mcsweeneys.net##.newsletter
```

Comment:
We have a common rule for that.
What you should do:
1. remove the rule
2. add the domain `mcsweeneys.net,` to the common rule under the comment `! ##.newsletter`

Reply:
done 👍